python/cudf/cudf/tests/test_json.py

```diff
@@ -618,3 +618,16 @@ def test_json_nested_lines(data):
     # In the second test-case:
     # Pandas omits "f1" in first row, so we have to enforce a common schema
     assert df.to_arrow().equals(pa.Table.from_pandas(pdf))
+
+
+def test_json_nested_data():
+    json_str = (
+        R'[{"0":{},"1":[],"2":{}},{"1":[[""],[]],"2":{"2":""}},'
+        R'{"0":{"a":"1"},"2":{"0":"W&RR=+I","1":""}}]'
```

Comment:
Do these need to be raw strings (if I got the name right)?

Reply:
Don't need to be raw strings. Now, they're just strings 🙂